.gitignore

```diff
@@ -15,3 +15,4 @@
 /dist
 /resource_windows_*.syso
 /versioninfo.json
+/.idea
```

Comment:
Please remove this, user-specific configuration should go in your global `.gitignore` file.